### DISCUSSION TITLE
What is the typical process in which the manifest is updated when a new version of an app is released?

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [ScoopInstaller/Extras#14303] [Q&A]
When a new version of an app is released, what is the typical process in which the corresponding Scoop manifest is updated?  Or is there no typical for the `main` and `extras` buckets?

Are most apps assigned to specific manifest maintainers, or is it more like "hey, I noticed this app was recently updated, so I'll update the Scoop manifest for everyone"?

Are any of the manifests updated by automated means (for better or worse)?

**Comment by danielpauldd** [comment]
The most manifest contains an checkver and autoupdate section
example:    https://github.com/ScoopInstaller/Extras/blob/master/bucket/010editor.json

```
...
     "checkver": {
        "url": "https://www.sweetscape.com/download/previous/",
        "regex": "Download 010 Editor v([\\d.]+)"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://download.sweetscape.com/010EditorWin64Portable$version.zip"
            },
            "32bit": {
                "url": "https://download.sweetscape.com/010EditorWin32Portable$version.zip"
            }
        }
```
There are Github actions configured that use this infomation to update the manifest files.
